web/packages/teleterm/src/ui/Search/pickers/ActionPicker.tsx

```diff
@@ -399,6 +395,49 @@ export function KubeItem(props: SearchResultItem<SearchResultKube>) {
   );
 }
 
+export function NoResults(props: {
+  inputValue: string;
+  clusters: tsh.Cluster[];
+}) {
+  if (props.inputValue.length > 0) {
+    const excludedClustersCopy = getExcludedClustersCopy(props.clusters);
+    return (
+      <EmptyListCopy>
+        <Text typography="subtitle1" color="text.primary">
+          No matching results found.
+        </Text>
+        {excludedClustersCopy && (
+          <Item Icon={icons.Info} iconColor="text.primary">
+            <Text typography="body1" color="text.primary">
+              {excludedClustersCopy}
```

Comment:
Could you also add a test to `SearchBar` which verifies that this text is displayed when some clusters are not connected? I already added a couple of tests there so it should be fairly straightforward to write this test following the pattern.

Reply:
Done